Hi.

This patch against 2.2-rc15 removes the only use of a pageflags macro.
The macro itself is removed in a forthcoming patch.

Regards,

Nigel

 prepare_image.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)
diff -ruNp 9000.patch-old/kernel/power/prepare_image.c 9000.patch-new/kernel/power/prepare_image.c
--- 9000.patch-old/kernel/power/prepare_image.c	2005-12-27 20:43:33.000000000 +1000
+++ 9000.patch-new/kernel/power/prepare_image.c	2005-12-26 17:13:02.000000000 +1000
@@ -108,7 +108,7 @@ static inline int header_storage_needed(
 	 sizeof(struct suspend_header) +
 	 sizeof(struct plugin_header) +
 	 (int) header_storage_for_plugins() +
-	 (PAGES_PER_DYN_PAGEFLAG << PAGE_SHIFT) +
+	 (dyn_pageflags_pages_per_bitmap() << PAGE_SHIFT) +
 	 num_plugins *
 	 	(sizeof(struct plugin_header) + sizeof(int)));
 



_______________________________________________
Suspend2-devel mailing list
Suspend2-devel@lists.suspend2.net
http://lists.suspend2.net/mailman/listinfo/suspend2-devel

